modules/migration-updating-deprecated-internal-images.adoc

```diff
@@ -46,6 +46,17 @@ $ podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false <registry_u
 $ podman pull <registry_url>:<port>/openshift/<image>
 ----
 
+. Scan the {product-title} 3 image for deprecated namespaces:
++
+[source,terminal]
+----
+oc get bc --all-namespaces --template='range .items
```

Comment:
If this command is intentionally multi line and a $ prompt would break the copy/clipboard button then please ignore this comment, but this command otherwise might need a $ prompt at the beginning:
```suggestion
$ oc get bc --all-namespaces --template='range .items
```

Reply:
@dymurray @midays What do you say? OK to start with a $ ?

Reply:
Yup sounds good